content/en/docs/setup/production-environment/tools/kubeadm/enable-disable-dual-stack.md

```diff
@@ -0,0 +1,147 @@
+---
+reviewers:
+- sig-cluster-lifecycle
+- neolit123
+- sftim
+title: Enable Or Disable Dual-Stack via kubeadm
+feature:
+  title: Enable Or Disable Dual-Stack via kubeadm
```

Comment:
as mentioned here we should make the title more generic:
> Dual-stack support with kubeadm

https://github.com/kubernetes/website/pull/26675/files#r583667883

and file rename `dual-stack-support.md`